ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/SessionUtil.java

```diff
@@ -34,6 +35,17 @@
   private SessionUtil() {
   }
 
+  static void closeSilently(
+      final ServerWebSocket webSocket,
+      final CloseCodes code,
+      final String message) {
+    try {
+      webSocket.close((short) code.getCode(), message);
+    } catch (final Exception e) {
+      LOG.info("Exception caught closing websocket", e);
+    }
+  }
+
   static void closeSilently(
```

Comment:
With the changes in this PR, the only usages of this method are in `SessionUtilTest.java`. Can we update the tests there to test the new `closeSilently()` instead, and remove this old method?

Reply:
+1 This is actually removed in a later PR